client/state/themes/reducer.js

```diff
@@ -227,6 +226,10 @@ export const queries = ( () => {
 		[ THEMES_RECEIVE ]: ( state, { siteId, themes } ) => {
 			return applyToManager( state, siteId, 'receive', true, themes );
 		},
+		[ THEME_REQUEST_FAILURE ]: ( state, { siteId, themeId, error } ) => {
+			const theme = [ { id: themeId, error } ];
+			return applyToManager( state, siteId, 'receive', true, theme );
+		},
```

Comment:
😠 Seriously, let's not do that. I have no idea what `QueryThemeManager` is going to do with that, but I don't imagine it will be good.